common/build.gradle.kts

```diff
@@ -1,2 +1,32 @@
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+
 group = rootProject.group
 version = rootProject.version
+
+val ideaVersion: String by project
+
+plugins {
+    id("org.jetbrains.intellij.platform.module")
+}
+
+intellijPlatform {
+    instrumentCode = false
+}
+
+repositories {
+    intellijPlatform {
+        defaultRepositories()
+        jetbrainsRuntime()
+    }
+}
+
+dependencies {
+    intellijPlatform {
+        intellijIdeaCommunity(ideaVersion, useInstaller = false)
+        jetbrainsRuntime()
+        bundledPlugins("com.intellij.java", "org.jetbrains.kotlin")
+        testFramework(TestFrameworkType.Bundled)
+    }
```

Comment:
Unfortunately, I don't know a proper way how not to duplicate these lines if do everything as expected, i.e., not apply `org.jetbrains.intellij.platform` at the top-level where it's unnecessary, and use `org.jetbrains.intellij.platform.module` plugin for `common` module since `common` is not a plugin itself but just a part of it